akka-actor/src/main/resources/reference.conf

```diff
@@ -1052,6 +1052,18 @@ akka {
       async-dns {
         provider-object = "akka.io.dns.internal.AsyncDnsProvider"
 
+        # Parameters set upper bound of the caching duration for dns records
+        # Actually, caching duration in successful case(dns-resover has found dns record)
+        # is the smallest value from positive-ttl parameter and ttl value in the dns record
+        # In negative case(dns-resolver hasn't found any dns records),
+        # caching duration equals to negative-ttl parameter
+        # Possible values:
+        # forever: cache forever
+        # never: no caching
+        # n [time unit]: positive timeout with unit, for example 30s
+        positive-ttl = forever
```

Comment:
I think this chunk of doc now is a bit contradictory (first explains its the smallest for positive-ttl but then says forever means cache forever) so better put a chunk per setting with more correct information:

Suggestion
```
# Set upper bound for caching successfully resolved dns entries
# if the DNS record has a smaller TTL value than the setting that
# will be used. Default is to use the record TTL with no cap.
# Possible values:
# forever: always use the minimum TTL from the found records
# never: never cache 
# n [time unit] = cap the caching to this value
positive-ttl = forever

# Set how long the fact that a DNS record could not be found is
# cached. If a new resolution is done while the fact is cached it will
# be failed and not result in an actual DNS resolution. Default is
# to never cache.
# Possible values:
# never: never cache
# forever: cache a missing DNS record forever (you probably will not want to do this)
# n [time unit] = cache for this long
negative-ttl = never
```